src/plugins/layout/stateprovider.ts

```diff
@@ -129,3 +124,34 @@ export function createGenericLayoutState(node: Node, tag: string): ILayoutState
   };
   return layoutState;
 }
+
+export function shouldIgnoreNode(node: Node, shadowDom: ShadowDom): boolean {
+  let shadowNode = shadowDom.getShadowNode(getNodeIndex(node));
+  let ignore = false;
+  switch (node.nodeType) {
+    case Node.ELEMENT_NODE:
+      let tagName = (node as Element).tagName;
+      if (tagName === "SCRIPT" || tagName === "META") {
```

Comment:
1. Make the set of nodes we ignore configurable. Default to these.
2. Why are we not sending these back now? At least he <meta> seems useful to have. 
3. It would be useful to send back script tags but not the children. Could we do that?